autogen/oai/openai_utils.py

```diff
@@ -504,11 +522,11 @@ def config_list_from_json(
         if os.path.exists(env_str):
             # It is a file location, and we need to load the json from the file.
             with open(env_str, "r") as file:
-                json_str = file.read()
+                json_or_yaml_str = file.read()
         else:
             # Else, it should be a JSON string by itself.
-            json_str = env_str
-        config_list = json.loads(json_str)
+            json_or_yaml_str = env_str
```

Comment:
If the yaml dependency is not installed, we can fall back to use `json` module. We can do this by checking for yaml import error during import, and use that information to choose whether to use yaml load or json load.

Reply:
oh, it sounds good